buf.gen.yaml

```diff
@@ -1,6 +1,4 @@
 version: v1
 plugins:
-  - remote: buf.build/prost/plugins/prost:v0.1.3-2
-    out: src/gen/
-  - remote: buf.build/prost/plugins/tonic:v0.1.0-2
```

Comment:
Out of curiosity, why are we no longer generating client/server code?

Reply:
Sorry this slipped my mind completely, we're not currently using it and at one point I thought it was the source of the generic component issue. I'll try adding it back and seeing what happens

Reply:
alas, it still doesn't work. Should we maybe make a ticket to bring it back once [this issue](https://github.com/neoeinstein/protoc-gen-prost/issues/61) is finally resolved? Micro-RDK doesn't use this for server-side code and I'm not aware of anyone using client-side Rust to interact with Viam